### DISCUSSION TITLE
友情链接

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#41] [Comments]
# 友情链接

## 如何交换友链？

1. 成为我的朋友，我们应该由一个初步的了解
2. 站点内容不要太水，最好能够保证一年至少更新一次
3. 告诉我你的站点信息，包括名称、url、logo和一个简短的描述
4. 在你的站点上增加指向本站的友链，然后想办法联系到我

## 本站信息

- Name: zu1k
- BIO: A boy dreaming of traveling around the world.
- URL: https://zu1k.com
- Logo: https://zu1k.com/images/logo.jpg

https://zu1k.com/links/

**Comment by hiifong** [comment]


**Reply by zu1k** [reply]
你好朋友，其实如何交换友链中第一条才是最重要的: `成为我的朋友，我们互相应该有一个初步的了解`，我们可以在其他社交媒体增加交流，合适时机我会添加友链

**Comment by Logistic98** [comment]
大佬您好，可以加个友链吗？
{
     "title": "Quantum Bit",
     "desc": "Never stop searching, even if only for a brief flash of light.",
     "logo": "https://www.imiao.top/avatar.png",
     "link": "https://www.imiao.top"
}
